packages/mobile/src/components/lineup/types.ts

```diff
@@ -111,4 +111,13 @@ export type LineupProps = {
    * another VirtualizedList.
    */
   listKey?: string
-} & Pick<ScrollViewProps, 'showsVerticalScrollIndicator'>
+
+  /**
+   * When `true` don't load more while lineup status is `LOADING`.
```

Comment:
do we ever want to have this false?

Reply:
agreed, we added the status check thinking we'd always want to do that. It ended up breaking a few lineups though: trending month/all-time, and reposts. just to be safe added this config in to solve the specific case for trackscreen. we should prob get to the bottom of this though. happy to do that now if you think its worth it

Reply:
gotcha gotcha. non-blocking for now! probably lower pri